models/users.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const userSchema = new mongoose.Schema({
+  password: { type: String, required: true },
+  email: { type: String, unique: true, index: true, required: true },
+  firstName: { type: String, required: true },
+  lastName: { type: String, required: true },
+  resetToken: { type: String },
+  expireToken: { type: Date },
```

Comment:
Tokens should not be stored on the user, is this a temporary token that gets removed after reset?